attribute_types.go

```diff
@@ -463,6 +464,9 @@ func (ctr *Counter) unmarshal(attr netfilter.Attribute) error {
 			ctr.Packets = iattr.Uint64()
 		case ctaCountersBytes:
 			ctr.Bytes = iattr.Uint64()
+		case counterType(ctaHelp):
```

Comment:
This is clever, but `CTA_HELP` happens to be the same value as  `CTA_COUNTERS_PAD`, which we should use instead. `CTA_HELP` has no meaning in this scope.